pyvista/plotting/scalar_bars.py

```diff
@@ -138,63 +138,77 @@ def add_scalar_bar(self, title='', mapper=None, n_labels=5, italic=False,
             Number of labels to use for the scalar bar.
 
         italic : bool, optional
-            Italicises title and bar labels.  Default False.
+            Italicises title and bar labels.  Default ``False``.
 
         bold  : bool, optional
-            Bolds title and bar labels.  Default True
+            Bolds title and bar labels.  Default ``True``.
 
         title_font_size : float, optional
             Sets the size of the title font.  Defaults to ``None`` and is sized
-            according to ``pyvista.global_theme``.
+            according to :attr:`pyvista.themes.DefaultTheme.font`.
 
         label_font_size : float, optional
             Sets the size of the title font.  Defaults to ``None`` and is sized
-            according to ``pyvista.global_theme``.
+            according to :attr:`pyvista.themes.DefaultTheme.font`.
 
         color : string or 3 item list, optional
-            Either a string, rgb list, or hex color string.  Defaults to white.
-            For example:
+            Either a string, rgb list, or hex color string.  Default
+            set by :attr:`pyvista.themes.DefaultTheme.font`.  Can be
+            one of the following:
 
             * ``color='white'``
             * ``color='w'``
             * ``color=[1, 1, 1]``
             * ``color='#FFFFFF'``
 
         font_family : {'courier', 'times', 'arial'}
-            Font family.  Default is set by ``pyvista.global_theme``.
+            Font family.  Default is set by
+            :attr:`pyvista.themes.DefaultTheme.font`.
```

Comment:
I can't get the links to go that deep since `font` is an attribute of DefaultTheme.  I could just reference `_Font`, but I figure that the user can figure this out, especially as we provide examples.